docs/rst/tutorials/cloud_tutorial.rst

```diff
@@ -0,0 +1,166 @@
+.. _vulcanexus_cloud_tutorial:
+
+Vulcanexus Cloud
+================
+
+.. contents:: Table of Contents
+    :depth: 2
+    :local:
+
+Background
+----------
+
+This walk-through tutorial sets up both a *Kubernetes* (*K8s*) network and a local environment in order to establish communication between a pair of ROS nodes, one sending messages from a LAN (talker) and another one (listener) receiving them in the Cloud.
+Cloud environments such as container-oriented platforms can be connected using *eProsima DDS Router*, and thus, by launching a *DDS Router* instance at each side, communication can be established.
+
+.. figure:: /rst/figures/cloud_tutorial/ddsrouter_overview_wan.png
+
+Prerequisites
+-------------
+
+Ensure that the Vulcanexus installation includes the cloud and the ROS 2 demo nodes package (it is suggested to use ``vulcanexus-galactic-desktop``).
+
+Also, remember to source the environment in every terminal in this tutorial.
+
+.. code-block:: bash
+
+    source /opt/vulcanexus/galactic/setup.bash
```

Comment:
```suggestion
    source /opt/vulcanexus/$VULCANEXUS_DISTRO/setup.bash
```

Reply:
The tutorial is specifically for Vulcanexus galactic and it is going to be merged in the `galactic` branch. The idea is that the user can copy/paste the commands when possible to run the tutorial. The environment variable `$VULCANEXUS_DISTRO` is just set by running the previous command, so applying your change will make the command to fail. I do not think we should make this change.

Reply:
I think I suggested something similar somewhere, but you're right that `source setup.bash` is the command that will set `$VULCANEXUS_DISTRO` in the first place, so please ignore my suggestion regarding this as well.

Reply:
Ok I see, I thought the variable would be somehow exported when installing the package.

Reply:
You suggested to change some `$ROS_DISTRO` for `$VULCANEXUS_DISTRO` which made some sense. The thing is that this is assuming that the `ubuntu-vulcanexus-cloud` Docker container from which we are building, has already set these environment variables. Maybe, instead of trying to generalize, it makes more sense to use `galactic` in all these files concerning this Cloud toturial.

Reply:
Yeah if this tutorial is specifically intended for `galactic`, then I totally agree we could forget about using the env variables and just write the version explicitly.

Reply:
If we are talking about `ubuntu-vulcanexus-*` docker images, then the variable should be always set. What I mean is that when you run `docker run -it ros2:galactic /bin/bash`, you do not need to `source `/opt/ros/galactic/setup.bash` as is it done already. I think it is fair to assume that our images, when run, will source the vulcanexus environment. In fact, that is a requirement IMO.